### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lucasnestrojil**
### Update readme.md:
   - Added possible creation scripts 
   - Added attached libraries 
   - Added about section

## Trello Card
https://trello.com/c/xj6eev6J/220-documentar-nuevas-opciones

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean proyect with the most used libraries of a react proyect
+
+- ### Customized
+
+Choose your preferred libraries of your proyect and create a new react proyect with standarts components, default screens, apis servicies and common utils functions.
```
Comment (damfinkel) [inline README.md]: `api services`

File: README.md [hunk 2]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+|----|-----------|
+|[react]()| v16.6.3
+
+
+- ### Clean
+
+|Name|Description|
+|----|-----------|
+|[eslint-config-wolox](https://github.com/Wolox/eslint-config-wolox)| Wolox eslint standard rules.
+|[eslint-config-wolox-react](https://github.com/Wolox/eslint-config-wolox-react)| Wolox eslint reactJs rules.
```
Comment (damfinkel) [inline README.md]: `React rules`

File: README.md [hunk 3]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+|[prettier]()| Code formatter.
+|[react-hot-loader](https://www.npmjs.com/package/react-hot-loader)| Allow live reloaded without the loss of state
```
Comment (damfinkel) [inline README.md]: `Allow live reload without loss of state`

File: README.md [hunk 4]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+|[postcss](https://www.npmjs.com/package/postcss)| Is a tool for transforming styles with JS plugins.
```
Comment (damfinkel) [inline README.md]: `Tool for transforming`

File: README.md [hunk 5]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+|[enzyme](https://www.npmjs.com/package/enzyme)| Is a JavaScript Testing utility for React that makes it easier to test your React Components' output
```
Comment (damfinkel) [inline README.md]: `JavaScript testing utility`

File: README.md [hunk 6]
```diff
@@ -59,3 +139,9 @@ One possible way to fix it is the following:
 - Reinstall it: `npm install -g yo`
 
 WARNING: Be careful when setting these variables to set them to the correct node version or you may completely break npm. If you accidentaly do break it, you must uninstall node and npm, reinstall them and start over. 
+
+## About
+
+This repository is maintained by everyone at  [Wolox](https://wolox.co).
```
Comment (damfinkel) [inline README.md]: remove space before [Wolox]

File: README.md [hunk 7]
```diff
@@ -59,3 +139,9 @@ One possible way to fix it is the following:
...
+
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
```
Comment (damfinkel) [inline README.md:151]: Let's add a copyright, copy the one in https://github.com/Wolox/redux-recompose

**Review by damfinkel — CHANGES_REQUESTED** [review]
Nice work, I like how it's looking 👍

File: README.md (hunk 4 above)
Comment (lucasnestrojil) [reply README.md]: `tool` with capital letter?

File: README.md [hunk 8]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean proyect with the most used libraries of a react proyect
+
+- ### Customized
+
+Choose your preferred libraries of your proyect and create a new react proyect with standarts components, default screens, api services and common utils functions.
```
Comment (pabloferro) [inline README.md]:
```suggestion
Choose your preferred libraries of your project and create a new react project with standard components, default screens, api services and common utils functions.
```

File: README.md [hunk 9]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+
+|Name|Description|
+|----|-----------|
+|[eslint-config-wolox](https://github.com/Wolox/eslint-config-wolox)| Wolox eslint standard rules.
+|[eslint-config-wolox-react](https://github.com/Wolox/eslint-config-wolox-react)| Wolox eslint React rules.
+|[prettier]()| Code formatter.
+|[react-hot-loader](https://www.npmjs.com/package/react-hot-loader)| Allow live reloaded without loss of state
+|[postcss](https://www.npmjs.com/package/postcss)| Is a tool for transforming styles with JS plugins.
+|[enzyme](https://www.npmjs.com/package/enzyme)| Is a JavaScript testing utility for React that makes it easier to test your React Components' output
+|[react-test-renderer](https://www.npmjs.com/package/react-test-renderer)|  Provides an experimental React renderer that can be used to render React components to pure JavaScript objects
```
Comment (pabloferro) [inline README.md]:
```suggestion
|[react-test-renderer](https://www.npmjs.com/package/react-test-renderer)| Provides an experimental React renderer that can be used to render React components to pure JavaScript objects
```

File: README.md [hunk 10]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+|[chalk](https://www.npmjs.com/package/chalk)| Terminal Styling
+|[env-cmd](https://www.npmjs.com/package/env-cmd)| Node program for executing commands using an environment from an env file.
+|[aws-deploy-script-fe](https://www.npmjs.com/package/aws-deploy-script-fe)| AWS script for deploying your frontend applications.
+|[node-sass]()| Provides binding for Node.js to LibSass, the C version of the popular stylesheet preprocessor, Sass.
+|[react-dom]()| Serves as the entry point to the DOM and server renderers for React
```
Comment (pabloferro) [inline README.md]: Do we have links for these?

**Review by pabloferro — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (damfinkel) [reply README.md]: I mean delete "Is a", just leave `| Tool for transforming styles with JS plugins`

File: README.md [hunk 11]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
 ./react-bootstrap/run.sh -l
 ```
 
+## Bootstrap options
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean project with the most used libraries of React
```
Comment (damfinkel) [inline README.md]: `Create a clean project using crate-react-app as baseline and with some small addition to its configuration`

File: README.md [hunk 12]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+- ### Clean
+
+|Name|Description|
+|----|-----------|
+|[eslint-config-wolox](https://github.com/Wolox/eslint-config-wolox)| Wolox eslint standard rules.
+|[eslint-config-wolox-react](https://github.com/Wolox/eslint-config-wolox-react)| Wolox eslint React rules.
+|[prettier]()| Code formatter.
+|[react-hot-loader](https://www.npmjs.com/package/react-hot-loader)| Allow live reloaded without loss of state.
+|[postcss](https://www.npmjs.com/package/postcss)| Is a tool for transforming styles with JS plugins.
+|[enzyme](https://www.npmjs.com/package/enzyme)| Is a JavaScript testing utility for React that makes it easier to test your React Components' output.
```
Comment (damfinkel) [inline README.md]: `| JavaScript testing utility for React that makes it easier to test your React Components' output`

**Review by damfinkel — COMMENTED** [review]
3 last comments

File: README.md [hunk 13]
```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
...
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean project with the most used libraries of React
+
+- ### Customized
+
+Choose your preferred libraries for your project and create a new React project with standards components, default screens, api services and common utils functions.
```
Comment (pabloferro) [inline README.md]:
```suggestion
Choose your preferred libraries for your project and create a new React project with standard components, default screens, api services and common utils functions.
```

File: README.md (hunk 11 above)
Comment (pabloferro) [inline README.md]:
```suggestion
Create a clean project using create-react-app as baseline and with some small additions to its configuration
```
Capi's comment has a typo in crate-react-app 👀

**Review by pabloferro — CHANGES_REQUESTED** [review]

**Review by pabloferro — APPROVED** [review]

**Review by damfinkel — APPROVED** [review]

**Review by carlosbollero — APPROVED** [review]

**Pull request merged**